build/charts/antrea/values.yaml

```diff
@@ -67,6 +67,26 @@ ovs:
   # configuration).
   hwOffload: false
 
+secondaryNetwork:
+  # -- Enable secondary network configuration in Antrea.
+  enable: false
+  ovs:
+    # -- Secondary network OVS integration bridge name.
+    ovsIntegrationBridgeName: "br-secnet-int"
+    # -- Secondary network OVS transport bridge name.
+    ovsTransportBridgeName: "br-secnet-trans"
+    # -- Datapath type to use for the OpenVSwitch bridge created by Antrea. Supported values are:
+    # - system
+    # - netdev
+    # -- 'system' is the default value and corresponds to the kernel datapath. Use 'netdev' to run
+    # OVS in userspace mode. Userspace mode requires the tun device driver to be available.
+    ovsDatapathType: "system"
+    # -- Name of the OVS patch port which connects the integration and transport bridge.
+    ovsPatchPort: "br-secnet-patch0"
```

Comment:
do we need to repeat `ovs` in all the field names given that they are already in a dedicated `ovs` section?

Reply:
I would agree we can remove the `ovs` prefix as these parameters are already under the `ovs` subsection.